%
%                       This is a basic LeTeX Template
%                       for the First Year PhD literature review 
\documentclass[a4paper,12pt]{article}

\usepackage{head,fullpage}     % Add local fullpage and head macros
\usepackage{graphicx}
\usepackage{subcaption} % Add graphicx pachage with pdf flag (must use pdflatex)
\usepackage{datetime}
\usepackage{float}

\renewcommand{\dateseparator}{-}
\parindent=0pt          %  Switch off indent of paragraphs 
\parskip=5pt            %  Put 5pt between each paragraph  


\let\oldquote\quote
\let\endoldquote\endquote
\renewenvironment{quote}[2][]
  {\if\relax\detokenize{#1}\relax
     \def\quoteauthor{#2}%
   \else
     \def\quoteauthor{#2~---~#1}%
   \fi
   \oldquote}
  {\par\nobreak\smallskip\hfill(\quoteauthor)%
   \endoldquote\addvspace{\bigskipamount}}
   
   
%
%                       This section generates a title page
%                       Edit only the sections indicated to put
%                       in the project title, your name, supervisor,
%                       project length in weeks and submission date
%
\begin{document}
\begin{minipage}[b]{110mm}
        {\Huge\bf School of Physics \\and Astronomy
        \vspace*{17mm}}
\end{minipage}
\hfill
\begin{minipage}[t]{40mm}               
        \makebox[40mm]{
        \includegraphics[width=40mm]{crest}}
\end{minipage}
\par\noindent                                           % Centre Title, and name
\vspace*{2cm}
\begin{center}
        \Large\bf Modelling the growth of microbial populations in heterogeneous antimicrobial concentrations \\
        \Large\bf First Year Report and Literature Review
\end{center}
\vspace*{1.5cm}
\begin{center}
        \bf Patrick Sinclair\\                 % Replace with your name
        May 2018                          % Submission Date
\end{center}
\vspace*{5mm}
%
%                       Insert your abstract HERE
%                       
\begin{abstract}
	Spatial gradients of antimicrobial chemicals are ubiquitous in both modern medicine and industry, from bacterial biofilms caused by infection and the application of antibiotics, 
	to marine biofouling and the leaching of antimicrobials from specially designed shipping hull coatings.  
        This project aims to use stochastic computer modelling techniques to investigate how the presence of antimicrobial gradients can affect the growth and proliferation 
        of microbial populations in order to better understand how to minimise the formation and persistence of microbial biofilms.
\end{abstract}

\vspace*{1cm}

\vspace*{3cm}
Signature:\hspace*{8cm}Date:

\vfill
{\bf Supervisor:} Dr. Rosalind Allen                % Change to suit
\newpage
%                                               Through page and setup 
%                                               fancy headings
\setcounter{page}{1}                            % Set page number to 1
\footruleheight{1pt}
\headruleheight{1pt}
\lfoot{\small School of Physics and Astronomy}
\lhead{Literature Review}
\rhead{- \thepage}
\cfoot{}
\rfoot{Date: \ddmmyyyydate \today}
%
\tableofcontents                                % Makes Table of Contents
\pagebreak

\section{Background}

% \begin{itemize}
%  \item Antibiotic resistance - its prevalence, not understood well
%  \item overview of gradients frm martin paper - then mention a source of gradients is biofilms
%  \item Biofilms - what are they (mention polymers perhaps)
%  \item section on industrial applications - ship hulls etc
% \end{itemize}

Since their discovery in the early 1900s \cite{bioref:first-antibiotics}, antibiotics have shaped modern medicine,
and indeed modern society as a whole.  Yet despite their prevalence, with over 260,000,000 courses of antibiotics prescribed 
in the USA alone each year \cite{bioref:antibiotics-usage-USA-2011}, very little is still known about the actual 
underlying pharmacodynamics, i.e., how these chemicals actually regulate the growth and death of bacteria.  This 
lack of understanding is becoming increasingly significant with the rising emergence of antibiotic resistance.  
There were over 58,000 deaths in newborns under the age of a year in India in 2013 due to drug-resistant 
strains of bacteria \cite{bioref:india-death-stats}, with experts predicting that the number of deaths from 
antibiotic-resistant bacteria will number in the millions by 2050 \cite{bioref:future-death-stats}.

Part of the work involved in this PhD aims to shed further light on how the application of antibiotics causes bacterial populations
to evolve and proliferate over time.  In particular, in the cases of when the applied antibiotic concentration has
the form of a gradient.  A considerable portion of antibiotic research has been performed under laboratory conditions
with constant, uniform antibiotic concentration \cite{bioref:Grasso-constant-antibtioic-concn}, however it has now become clear that the 
effects of spatial heterogeneity may be a major factor in the emergence of resistance 
and the efficacy of drugs \cite{bioref:Zhang-effects-of-antibio-grad}.  While these concentration gradients can simply arise 
due to scenarios such as diffusion throughout body tissue \cite{bioref:tissue-antibio-grads}, the scenario which is most pertinent
to this project is that of antibiotic concentration gradients in biofilms.

Biofilms arise when microbial organisms adhere irreversibly to a surface and then begin to secrete various polymers, which further aid
in surface and inter-microbial attachment \cite{bioref:biofilm-formation}, so that the microbial population create a ``slimey'' surface.  These structures are 
particularly problematic as bacteria found in biofilms are innately more resistant to applied antimicrobials and  it is also difficult to achieve sufficient drug 
penetration throughout the biofilm to adequately curtail microbial growth, which leads to an increase in the persistence of infections \cite{biofilms:Costerton-biofilms-persistent}.

The ability to inhibit and even prevent biofilm growth and formation not only has a multitude of medical applications, but has industrial applications as well.
In the shipping industry it is estimated that around 10\%, up to even 45\%, of all fuel consumption of large shipping vessels arises from overcoming the 
hydrodynamic drag caused by algal and other microbial biofilm formation on ship hulls below the water level \cite{bioref:biofilm-fuel-consumption}.  This not only has
economic influences, but also major environmental implications.  When compared to other nations, the shipping industry is the 7th largest 
producer of CO$_2$ on the planet \cite{bioref:shipping-CO2-nation}.  Therefore, research into how these marine biofilms form and develop is 
incredibly important.  To this end, this PhD also involves collaboration with AkzoNobel, an industrial paint company, which will entail modelling the formation of biofilms 
on the exterior of ship hulls in the presence of antimicrobial chemical gradients.

Recently, several physical methods of reducing marine biofouling have been developed, which range from physical coatings that inhibit microbial
attachment due to their topography \cite{bioref:non-toxic-antifouling-strat} to usage of ultraviolet radiation \cite{bioref:UV-biofilm-repellent}.
However, the most widespread technique is that of anti-microbial paints which are applied to the boat hulls and then leach various antimicrobial
compounds over time which inhibit biofouling \cite{bioref:antimicro-paint-desc}.  It is this latter method which is of relevant interest to this project, 
as it has analogies with the application of antibiotics to bacterial biofilms.

This project will involve creating computer simulations of a range of scenarios where microbial populations experience gradients of antimicrobial chemicals, and will
investigate how these populations develop over time.  Work so far has included differing types of growth-rate dependent
antibiotics and populations with heterogeneous species and resistance distributions.




% Outline\cite{bioref:KUMAR19989} the background of your subject area including the key initial
% References \cite{jr:ashkin} and reference\cite{ob:Costerton1318} textbooks \cite{ob:bornwolf}. 
% Also include some of the more
% readable articles in popular science journals \cite{jr:dholakia},
% and, where appropriate, standard textbooks \cite{ob:hechtoptics}.
% 
% The exact length of this section will depend on your subject area,
% but will generally not exceed a page and will be aimed at the 
% {\it general scientific reader}. 

\section{Review of Background Bibliography}

% \begin{itemize}
% %   \item overview of antibiotics - how they work etc
% %   \item Causes of antibiotic crisis
% %   \item growth rate dependent antibiotics, $\beta$-lactams etc
% %   \item how biofilms form - the bacteria change when in a biofilm, why are they difficult to get rid of 
% %   \item overview of biofouling, current ship hull anti-microbes
% %   \item difference of films and fouls
% %   \item brief mention of macrofouling
% %   \item surface colonisation
% %   \item opposite gradient directions for films and fouls
% %   \item how biocidal paint works, diffusion coefficients etc
% %   \item mention the differnt algorithms possible for the sims, gillespie etc - and give overview of what we used
% %   \item as in, mention the rates, algorithm and description of why used
%  \end{itemize}

\subsection{Antibiotic gradients}
The issue of antibiotic resistance is one of the key issues plaguing modern science as of today.  As such, the field commands a 
large volume of dedicated research utilising a wide range of methods.  Ranging from experimental to theoretical techniques including both modelling and 
more in-depth simulation \cite{bioref:chait-interactions, bioref:Wang-treatment-tradeoff, bioref:Torella-optimal-drug-synergy}.  Current research is 
investigating a wide variety of factors which contribute to the development of resistances.  From mutational path lengths \cite{bioref:marvig-transmiss-lineage} to 
the synergistic effects of various antibiotics \cite{bioref:Liu-baicalin-synergy}.  However, the majority of these studies, including all studies referenced
so far in this section, are performed with constant and uniform concentrations of antibiotics.

While this situation tends to be more convenient for idealised in vivo experiments, many real-world in vivo scenarios do not have these conditions. Many naturally 
occurring structures which the drugs are intended to target, tend to not allow the drugs to fully permeate throughout
the region, creating gradients where the drug concentration can vary noticeably over space.  These gradients can arise in a variety of 
situations, from tissue \cite{bioref:minelli-peflox-penet} to bacterial biofilms \cite{bioref:Kim-biofilm-antibio-grad-2010}.

It is only in recent years that the effects of these spatial heterogeneities have been considered a serious influence on the evolution of resistance. 
In fact, models have been constructed which predict that antimicrobial gradients can actually accelerate the evolution of resistance \cite{bioref:Hermsen-source-and-sink}.
These models were inspired by an experiment conducted by Zhang et al. \cite{bioref:Zhang-effects-of-antibio-grad}.  In which they constructed a microfluidic device involving an array of 
several interconnected microhabitats with an antibiotic gradient of ciprofloxacin.  This gradient ranged from no discernible antibiotic concentration at the top of the array to 
a concentration of 10$\mu$g/ml at the bottom.  This concentration is around 200 times the minimum inhibitory concentration (MIC, i.e., the minimum necessary concentration of 
an antibiotic required to prevent observable growth of bacteria) of ciprofloxacin \cite{bioref:ciprofloxa-mic}.

The array was then inoculated in the central microhabitats with around $10^6$ wild type \textit{E. coli}.  Chemotaxis due to nutrient consumption 
then drove the bacteria towards the perimeter microhabitats.  Once resistant mutants had fixed, they then spread and propagated throughout the array,
as shown in Figure \ref{fig:Zhang-gradient-apparatus}.

\begin{figure}[h]
 \centering
 \includegraphics[width=8cm]{Zhang-microhab-gradient-cropped}
 \caption{The proliferation of the bacterial population when exposed to an antibiotic gradient.  The LHS shows the development
 of the population after the initial inoculation, and the RHS shows the development of an identical slide which was 
 inoculated with the resistant mutants. Zhang et al., 2011}
 \label{fig:Zhang-gradient-apparatus}
\end{figure}

To confirm that it was indeed the gradient which allowed for this enhanced development of resistance, Zhang et al. conducted a range of further experiments.
Firstly they eliminated the gradient by including ciprofloxacin at both ends of the array.  This uniform antibiotic concentration resulted in no growth
from the inoculated wild-type \textit{E. coli}, as can be seen in Figure \ref{fig:Zhang-gradient-growth-graph}.

\begin{figure}[h]
 \centering
 \includegraphics[width=10cm]{Zhang-microhab-gradient-growth-graph}
 \caption{The summed growth over the entire array for various scenarios.  The green diamonds are the initial experiment where the wild-type \textit{E. coli}
 were placed in the antibiotic gradient.  The red circles are the resistant mutants which were then used to re-inoculate an identically set up array.
 The blue triangles are the wild-type exposed to a uniform antibiotic concentration.  Zhang et al., 2011}
 \label{fig:Zhang-gradient-growth-graph}
\end{figure}


Zhang et al. then performed the experiment in a 96 well plate with a gradient present, but with the microhabitats now disconnected from one another, with discrete 
antibiotic concentrations in each well, ranging from low to high concentrations as in the previous array.  This also resulted in no resistance being developed,
as the growth of the bacterial colonies simply decreased as the concentration of ciprofloxacin increased, thereby implying that bacterial motility across the 
gradients is what is key to the emergence of resistance.  

% To confirm the effects of motility on resistance development, Zhang et al. then used a series of agar plates with the same gradients as the microhabitat arrays,
% but varied the initial population sizes.  Once they reached an initial size of $10^8$ bacteria, no motility was observed and as such growth only occurred in 
% the regions of the plate where the antibiotic concentration was below the MIC, and no clear resistant mutants emerged.
% 
% Zhang et al. then proceeded to investigate what the source of the resistant mutants were, whether they were simply the descendants of already-present
% mutants, or if it was indeed legitimate de novo mutation as a response to the applied antibiotic stress.  Zhang et al. reasoned that if resistance emerged due to 
% preexisting, albeit rare, mutants, then either growth would have occurred above the MIC in the 96 well plate (which did not occur), or by serial dilution of the 
% starting wild-type colony, they  an initial population density would eventually be reached which contained no resistant mutants, and therefore no growth would occur.


\begin{figure}[h]
 \centering
 \includegraphics[height=9.6cm]{F2-large}
 \caption{Bacterial growth over time in an antibiotic gradient up to 1 $\mu$g/ml of ciprofloxacin for various initial bacterial population sizes.  
 The initial sizes range from the order of $10^6$ to as low as $10^2$.  It can be seen that even at the lowest starting densities, resistance still emerges.
 Thus supporting the argument that resistance emerges due to de novo mutation and not preexisting mutants.  Zhang et al., 2011}
 \label{fig:Zhang-gradient-inital-pop-sizes}
\end{figure}


As can be seen in Figure \ref{fig:Zhang-gradient-inital-pop-sizes}, even the most diluted populations developed resistance when exposed to the antibiotic gradient.  
These results heavily support the proposal that the source of emergent resistance is due to de novo mutations and not from preexisting mutants distributed amongst the 
wild-type.  Zhang et al. offered the following explanation as to why antibiotic gradients allow for such prevalent opportunities for resistance to emerge. 

\begin{quote}{Zhang et al., 2011}
 A spatially complex environment may lead to an enhanced rate of evolution for two reasons.  First, if a stress gradient is imposed on a connected network of 
 populations, and if a mutant acquires some resistance to the local stress, the relative fitness of the mutant is increased if it moves to join a population
 exposed to even higher stress.  Second, because there are fewer individuals in the region of higher stress, the mutant can fix more quickly in the smaller
 population.
\end{quote}

\subsection{Modelling the effects of antibiotic gradients} \label{subsec:modelling-gradients}


As can be seen, there is experimental evidence supporting the notion that non-uniform drug distributions can accelerate the emergence of resistant organisms.  To gain 
a better understanding of whether this is always the case and also how the mutants emerge and propagate,  Greulich et al. (2012) 
\cite{bioref:PRL-drugGradients} constructed a simple computational model which investigated how a bacterial population evolved along pathways 
in genotype space when exposed to both uniform and non-uniform antibiotic concentrations.  It is this model which has also formed the basis for the other models
constructed in this project, although the work conducted with these models has focused less on evolution and more on colonisation along the antimicrobial gradient.

The model consisted of $L$ microhabitats interconnected in series with one another.  Each microhabitat had a concentration of antibiotic $c_i$ (where $i$ is the index of the 
microhabitat), and a carrying capacity $K$.  If the number of bacteria in the microhabitat was $\geq{K}$, then no bacterial growth could occur in this microhabitat until its population 
decreased below the carrying capacity.  

Each bacterium had a numeric genotype $m$, which they could mutate between with a probability $\mu$ 
and had a maximum value of $M$.  This genotype described the level of resistance the bacteria had to the antibiotic, with $m_2$ being more resistant than $m_1$ and so on.  
At each step in the simulation each bacteria could die or move to an adjacent microhabitat at constant rates $d$ and $b$, or replicate at a rate given by 

\begin{equation}
 R_{rep} = \phi_m(c_i)(1 - \frac{N_i}{K}).
 \label{eqn:R_rep}
\end{equation}

Where $N_i$ is the total number of bacteria present in microhabitat $i$ and $\phi_m(c_i)$ is the genotype and antibiotic dependent growth rate.  
This value decreases until the MIC for that particular genotype ($\beta_m$) is reached, after which that bacteria cannot replicate.  The simulation is 
initialised by placing $K$ bacteria of genotype $m=1$ in the first microhabitat and then allowing them to proliferate throughout the system.  To illustrate the effects 
of the antibiotic gradient, the simulations were performed under two different conditions; with a uniform antibiotic concentration ($c_i = c$) and with an exponentially 
increasing antibiotic concentration

\begin{equation}
 c_i = \exp(\alpha{i}) - 1.
 \label{eqn:c_i}
\end{equation}

Here $\alpha$ is a dimensionless constant which dictates the steepness of the antibiotic gradient, i.e. by how much it increases between microhabitats, and by extension, 
the concentration of antibiotic in the final microhabitat.


\begin{figure}[H]
 \centering
 \includegraphics[height=5.6cm]{greulich-geno-distbs}
 \caption{Distributions of bacterial genotypes throughout the series of interconnected microhabitats.  The thick blue line is the concentration of antibiotic per 
 microhabitat, and the colours represent the distributions of genotypes.  $m=1$ is blue, $m=2$ green, with $m=3, 4, 5$ shown by red, orange and olive respectively.  
 For the uniform concentration, any mutations are sporadic and randomly  distributed, however the exponential gradient shows clear emergence of resistance in a 
 consistent manner.  Greulich et al., 2012.}
 \label{fig:Greulich-geno-distbs}
\end{figure}

Once again, as shown in Figure \ref{fig:Greulich-geno-distbs}, the dynamics of how evolution emerges varies greatly between the uniform and non-uniform drug distributions.  
For the uniform case, mutations arise sporadically and proliferate randomly, with the system as a whole generally evolving from one genotype to the other.  However when 
exposed to the gradient, the genotypes tend to form ``stationary fronts'', with resistant mutants emerging at the tip of the colony, then quickly spreading to fill
the remaining space until the MIC for the current advancing genotype is reached, at which point the process repeats itself.  As Figure \ref{fig:Greulich-geno-distbs} 
illustrates, where the snapshots were taken after an equal amount of time had passed, the presence of the gradient causes the emergence of ``population waves'' of increasing 
resistance, which greatly reduces the time taken for resistance to emerge.


% To further investigate the effects of the gradients, Greulich et al. ran a series of simulations varying the overall concentration of the antibiotic for the 
% uniform case, and the steepness of the gradient for the non-uniform one and recorded the time taken ($\bar{\tau}$) for a fully resistant (i.e. $m = M$) mutant to arise.
% 
% 
% \begin{figure}[H]
%  \centering
%  \includegraphics[height=5.8cm]{greulich-time-til-resistance}
%  \caption{Time taken for a fully resistant ($m = M$) mutant to emerge for a variety of uniform drug concentrations $c$, and steepness of non-uniform drug gradient $\alpha$.  
%  The red dashed line shows the minimum time taken for the uniformly concentrated system to reach resistance.  Greulich et al., 2012.}
%  \label{fig:Greulich-time-til-resistance}
% \end{figure}
% 
% 
% This experiment produced an interesting result, as shown in Figure \ref{fig:Greulich-time-til-resistance}.  For the uniform case, $\bar{\tau}$ followed a simple inversely 
% proportional relation, which makes intuitive sense.  With a higher drug concentration present, there is also an increased pressure for resistance to emerge.  However 
% for the non-uniform case, the results are somewhat more involved.  For the smallest values of $\alpha$, it's seen that the presence of the gradient actually causes $\bar{\tau}$ 
% to be larger than in the uniform case, as there is much less pressure on the system for resistance to evolve.  Additionally, there seems to be an ``optimal'' value for $\alpha$, 
% above which $\bar{\tau}$ begins to increase.  This is due to the fact that for the higher values of $\alpha$, the regions which the lead mutants can fill get much narrower as 
% $\alpha$ increases, as seen in the snapshots included in Figure \ref{fig:Greulich-time-til-resistance}.  Thus reducing the available regions that the next generation of mutants 
% can arise from.  From this it appears that the contributions of antibiotic gradients on an evolving system are even more complex than first supposed.

\subsection{Growth rate dependency} \label{subsec:GR-dependency}

It has long been known that the growth rate of bacteria affects their susceptibility, with slow growing or metabolically inactive bacteria having a much higher tolerance than their 
fast growing counterparts \cite{bioref:Cozens-slow-bac-grow-1986}.  However Greulich et al., 2015 \cite{bioref:Greulich-growthDependentAntibiotics} proposed that this might not
exclusively be the case, and instead that certain ribosome-targeting antibiotics are more effective against fast-growing or slow-growing cells depending on the mechanism which the 
antibiotic uses to bind to the target cell's ribosomes.

For example, the antibiotic tetracycline binds reversibly to the bacteria's ribosomes and is more effective when the bacteria are in a fast-growing state.  In contrast, streptomycin 
binds irreversibly tot he bacterial ribosomes and is more suited for bacteria which are less metabolically active.  It's been shown that the ribosome content of a cell correlates with 
the cell's growth rate \cite{bioref:Bremer-ribosome-content-2008}, with fast-growing cells dedicating more of their resources to ribosome production than their slow-growing 
counterparts \cite{bioref:Scott-ribosome-content-2010}. 

Although there are numerous factors which can affect a cell's growth rate, from oxygen limitation \cite{bioref:Dalton-oxygen-gRate-1968} to pH levels \cite{bioref:Russell-pH-gRate-1980}, 
the main environmental influence is that of nutrient availability, which was the variable Greulich et al. varied in order to alter the growth rates of their bacteria.  The behaviour 
of the nutrient-limited growth rate can be described by the Monod equation \cite{bioref:Monod-equation-1949} 

\begin{equation}
 \mu(S) = \mu_{\max}\frac{S}{K' + S}.
 \label{eqn:Monod}
\end{equation}

Here $\mu(S)$ is the current growth rate of the bacteria in this environment, $\mu_{\max}$ is the maximum growth rate of the bacteria, $S$ is a measure of the nutrient concentration 
present and $K'$ is a constant which represents the nutrient concentration at which growth is half-maximal.  As described this equation, the growth rate of bacteria decreases non-linearly 
as nutrient availability decreases.


\subsection{Biofilms}

% \begin{itemize}
%  \item mention factors that affect formation, roughness etc
%  \item changes undergone by bacteria in the biofilm - polymer secretions etc
%  \item defense mechanisms from that paper - gradients caused by diffusions and changes in growth rates throughout the film
%  \item this section will probs be the most involved
% \end{itemize}

Bacteria which aggregate onto a surface forming a biofilm have been shown to be much harder to eradicate than their free-flowing planktonic bacterial 
counterparts \cite{bioref:Lewis-biofilm-riddle-2001}.  In fact, biofilm colonies can exhibit resistances to antibiotics 10-1000 times greater than their individual complements 
\cite{bioref:Anderson-innate-biofilm-resistances-2008}, making treatment plans an immensely more complicated affair.  Whilst it is still relatively unclear as to what exactly affords 
biofilms these defences, experiments conducted in laboratory conditions heavily suggest that it is indeed the biofilm itself which possesses these qualities, rather than any external 
influence from the host or environment \cite{bioref:Stewart-biofilm-resis-mechanisms-2002}.  Biofilms can form on almost any surface where there is a consistent influx of bacteria, algae  
or other microorganisms, ranging from medical devices \cite{bioref:Donlan-biofilms-medical-devices-2002} to the exterior of ship hulls \cite{bioref:Chambers-modern-antifoul-coatings-2006}.  

The formation of biofilms is a complex and little-understood process.  Whilst it was originally thought that biofilms were formed by single organisms attaching to a surface, leading to 
the formation of micro-colonies and then 3D structures \cite{bioref:Monds-biofilm-formation-2009}, there is also now evidence that pre-existing bacterial aggregates can also 
seed the formation of biofilms \cite{bioref:hall-stooley-biofilm-clumps-2005}, the differing shape and composition of which in turn can influence the final established biofilm 
\cite{bioref:Xavier-biofilm-framework-multiD-modelling-2005}, making the developmental paths of biofilms increasingly hard to predict.

Regardless of how they come to be, biofilms have a range of features which distinguish them apart from a simple group of bacteria in close proximity to one another. One key trait 
that marks them apart is that of the inter-cellular matrix.  When bacteria or algae aggregate into a biofilm, they secrete a variety of macromolecules such as exopolysaccharides, DNA and 
protein fragments \cite{bioref:Whitechurch-biomatrix-components-2002}, which allows the microorganisms involved to adhere to both surfaces and one another.

This matrix forms just one of the biofilm's innate defence mechanisms.  Although its composition is mainly water, perhaps up to 97\% in some cases 
\cite{bioref:Zhang-biofilm-water97pc-1998}, the organic material present in it may act as a diffusion barrier to the applied antibiotic, preventing it from fully penetrating 
the biofilm.  On average, the diffusion coefficient of an administered antibiotic in a biofilm matrix is roughly 40\% that in pure water 
\cite{bioref:Stewart-biofilm-diffusion-coeff-1998}.  However it is not thought that this reduction in antibiotic mobility alone is enough to explain the levels of resistance which 
biofilms exhibit.  

Another feature of biofilms which may explain their durability is that of the formation of microenvironments within their structure \cite{bioref:Wimpenny-micrograds-1995}.  Factors such 
as oxygen limitation and poor nutrient diffusion throughout the biofilm matrix creates regions where the growth rate is substantially reduced compared to the outer sections.  As 
mentioned previously, the efficacy of many antibiotics are dependent on growth rate \cite{bioref:Field-gr-depend-antib-2005}, this therefore causes many antibiotics to only be effective 
against specific sections of a biofilm, rather than the structure as a whole.  

An example of which can be see in Figure \ref{fig:Pamp-dead-innards-biofilms}, taken from an experiment by Pamp et al., 2008 
\cite{bioref:Pamp-biofilm-internal-2008}.  Here they applied the antibiotic colistin to a biofilm of \textit{P. aeruginosa} bacteria to show that it is bacterial metabolic activity which 
contributes to the tolerance of colistin.  As shown in Figure \ref{fig:Pamp-dead-innards-biofilms}, the metabolically active cells on the periphery of the biofilm are able to tolerate 
the application of the antibiotic.  However the metabolically inactive cells towards the centre are unable to develop this tolerance, resulting in their death.  These multitude of 
factors contributing to the persistence of biofilms make further research into how they develop over time essential.  As despite their prevalence, surprisingly little is still known 
about their inner workings.

\begin{figure}[H]
 \centering
 \includegraphics[height=5.8cm]{Pamp-dead-biof-inside}
 \caption{The effects of the antibiotic colistin on a biofilm consisting of \textit{P. aeruginosa} bacteria.  Colistin is an antibiotic which targets slow-growing, metabolically 
 inactive cells, as can be seen, it is therefore more effective against the internal regions of the biofilm.  Live cells are shown in green, dead cells in red.}
 \label{fig:Pamp-dead-innards-biofilms}
\end{figure}


\subsection{Antimicrobial coatings}

Whilst the ability to eradicate, or at the very least suppress biofilm growth has numerous medical applications, it is also of notable economic and environmental consequence.  Marine 
biofouling, where a variety of micro and macroorganisms adhere to the exterior of a ship's hull, has extremely adverse effects on the efficiency of transportation.  Due to the 
contributions of the additional weight and surface roughness from marine biofouling, fuel consumption can increase by up to 45\% \cite{bioref:biofilm-fuel-consumption}.  This in 
turn affects the cost of these sea voyages, where a one-way trip between San Francisco and Yokohama has seen a price increase of 77\% due to the impact of 
biofouling \cite{bioref:Abbot-biofoul-price-2000}.

The process of marine biofouling begins to occur within minutes of a surface being exposed to a marine environment \cite{bioref:Callow-biofoul-onset-time-1994}, with the formation 
of a layer of proteins and polysaccharides not dissimilar to the intercellular matrix found in biofilms.  Following on from this, microorganisms such as bacteria and algae attach 
themselves to the surface and begin to grow.  Within 2-3 weeks, macroorganisms such as molluscs and seaweed have then adhered to this biofilm and a complex ecosystem has been formed 
\cite{bioref:Chambers-modern-antifoul-coatings-2006}.

It is the inhibition of the formation of the microorganism biofilm which will prove key to curtailing marine biofouling, as the majority of macrofouling is reliant on the presence 
of an established biofilm \cite{bioref:Abarzua-macrofoul-dependent-1995}.  As such, the majority of antifouling research is orientated towards the inhibition of the initial adhesion 
of marine bacteria, diatoms and other microorganisms.  Although numerous avenues of techniques have been pursued in this endeavour, the field of research relevant to this project 
is that of biocidal-containing paints.

These coatings consist of paints mixed with a variety of antimicrobial compounds which then leach said compounds into the surrounding marine atmosphere upon their submergence 
in seawater.  These coatings can be placed into two classes, of soluble or insoluble matrices, which depends on the manner of how the paints release their contained biocides.  
Coatings belonging to the insoluble matrix family contain an insoluble polymer framework, which persists after the soluble antimicrobials have been dissolved into the seawater.  This 
remaining porous structure is referred to as the ``leached layer'' \cite{bioref:Cao-bifoul-coatings-review-2011}.


%write about majority innate stuff, then diffusion . then do differing growth rates and gene alteration and that picture martin sent then done



% In this section detail the main supporting references
% and articles \cite{jr:block} for your intended area of research
% and, most importantly, your critical evaluation of their
% relevance.  Also where your subject draws from multiple 
% disciplines, do not forget to include key reference from
% each discipline, even if they are relatively old \cite{jr:dammann}. 
% 
% 
% This is the main part of your review and is the part that
% will be of use to you when preparing for your thesis. Here try
% and identify as many of the key references as possible, and enter then
% into a {\tt BibTeX} file that you will use later. Remember that recording
% the page number, titles and details of these 
% key articles {\it now} will save you hours of
% searching through {\em Web-of-Knowledge} the day before your
% submit your thesis!
% 
% This part should be written in standard scientific language, 
% aimed at the {\em experts in the field}. This is the main part of your first year report, and is 
% expected to be 10 pages in length.

\section{Progress to Date}

% \begin{itemize}
%  \item make several sections giving brief overview of them - PRL replication, growth dependent, multispecies 
%  \item mention paper
% \end{itemize}



\subsection{Replication of Greulich et al.}

The work undertaken so far in this PhD has comprised computer simulations investigating how microbial populations colonise along spatial concentration gradients of antimicrobial 
chemicals.  The progress made up until now in this project can be organised into several sections.  The initial few weeks of the project 
were spent replicating the results and techniques found in Greulich et al., 2012 \cite{bioref:PRL-drugGradients}.  Discussion was had
on the subject of which algorithm would be optimal for updating the system over time.  

Algorithms such as Gillespie \cite{bioref:Gillespie-algorithm} and $\tau$-leaping \cite{bioref:tau-leap-algorithm} were proposed, but eventually a simple Monte-Carlo style selection 
process as detailed in the supplementary material of Greulich et al., was decided upon.  The algorithm operated by selecting an individual cell at random and then summing together 
the migration, death and replication rates of the cell ($R_{mig}, R_{dea}, R_{rep}$), as detailed in section \ref{subsec:modelling-gradients}.  A random number $r$ was then chosen 
between 0 and $R_{\max}$, where $R_{\max} \geq R_{mig} + R_{dea} + R_{rep}$. Depending on the value of $r$, the cell would then either migrate, die, replicate or do nothing.  The time 
elapsed in the simulation was then increased by an increment $\Delta{t} = 1/(NR_{max})$.  

This method was found to be faster than the standard Gillespie algorithm as the number 
of calculations required for each iteration is small, and the rate at which an event occurs (i.e., the likelihood of $r < R_{mig} + R_{dea} + R_{rep}$) is relatively high, 
and on average around 25\%.  The only downside when compared to Gillespie is that the time elapsed is discretised into increments of $\Delta{t}$ rather than a continuous distribution.  
However the timescales that these simulations are performed for are much larger than $\Delta{t}$, so this is relatively inconsequential.  

The purpose of this was mainly to establish useful methods for future projects, intended to increase fluency and familiarity in the techniques and background 
theory required for the modelling of biological systems.  The results obtained from this body of work were rough, proof-of-concept illustrations, rather than the precise 
quantitative results obtained in the actual paper, intended more for comparative purposes to ensure the constructed model worked as intended.



\subsection{Modelling growth rate-dependant antibiotics}

The majority of the time spent on this project has been on the modelling of growth rate-dependant antibiotics, based on the 2015 paper by Greulich et al. 
\cite{bioref:Greulich-growthDependentAntibiotics} as mentioned in section \ref{subsec:GR-dependency}.  The foundation of this model was heavily borrowed from the one created in the 
previous section, but with the key addition of nutrients, which were used to modify the 
growth rate in lieu of the carrying capacity factor from the previous model.  Two simple functional forms for the MICs of fast-growing bacteria targeting antibiotics (FGBTA)
and slow-growing bacteria targeting antibiotics (SGBTA) were constructed as follows: for the fast-growth targeting antibiotics; 

\begin{equation}
 \beta_{FGT} =  10 - 9\frac{\mu(S)}{\mu_{\max}}
 \label{eqn:fast-growth-beta}
\end{equation}

and for the slow-growth targeting antibiotics;

\begin{equation}
 \beta_{SGT} = 1 + 9\frac{\mu(S)}{\mu_{\max}}.
 \label{eqn:slow-growth-beta}
\end{equation}


Here 

\begin{equation}
 \mu(S) = \frac{S}{K' + S}
\end{equation}

is the Monod equation as detailed in section \ref{subsec:GR-dependency}.  Each replication by a bacteria consumes nutrients, altering the growth rate and MIC
in the microhabitat.  As can be deduced from this, a concentration of 1 is required to impede bacterial growth for the FGBTA, but the required concentration 
increases as nutrients are consumed via replication, thus lowering the growth rate.  The inverse is therefore true for the SGBTA which require less antibiotic as the bacteria's 
growth rate decreases.

Therefore the full expression for the growth rate experienced by the bacteria is given by 

\begin{equation}
 R_{rep} = \max\{0, 1 - (\frac{c}{\beta})^2\}\frac{S}{K' + S}.
\end{equation}


As $\mu_{\max}$ is a constant representing the maximum concentration of nutrients in each microhabitat, the ratio $\frac{\mu(S)}{\mu_{\max}}$ 
will decrease over time with each successful replication.  Therefore for the bacteria exposed to the fast-growth targeting antibiotics, the tip of the advancing population will be 
more inhibited than the slow-growing bulk towards the rear of the colony, and vice versa for the slow-growth targeting antibiotics.  Additionally, as a form of ``control'' group, 
a series of simulations were also performed utilising 
growth-independent bacteria targeting antibiotics (GIBTA) whose MIC didn't vary with $S$ or $c$, but rather remained constant at all times.


\begin{figure}[H]
 \centering
 \begin{subfigure}[h]{0.49\textwidth}
 \includegraphics[width=\textwidth]{simple-slowGrowers-S_Vs_C-contours}
  \caption{SGBTA}
  \label{subfig:SGBTA-const_C-contours}
  \end{subfigure}
  \begin{subfigure}[h]{0.49\textwidth}
  \includegraphics[width=\textwidth]{simple-fastGrowers-S_Vs_C-contours}
  \caption{FGBTA}
  \label{subfig:FGBTA-const_C-contours}
 \end{subfigure}
\caption{Overall sizes of bacterial populations after 1000 time units have passed, for a variety of uniform antibiotic and initial nutrient concentrations.  
From this it can be seen that the SGBTA are less effective at inhibiting overall bacterial growth.}
\label{fig:const_C-popsize-contours}
\end{figure}



\begin{figure}[h!]
 \centering
 \begin{subfigure}[h]{0.49\textwidth}
 \includegraphics[width=\textwidth]{simple-slowGrowers-S_Vs_Alpha-contours}
  \caption{SGBTA}
  \label{subfig:SGBTA-alphagrad-contours}
  \end{subfigure}
  \begin{subfigure}[h]{0.49\textwidth}
  \includegraphics[width=\textwidth]{simple-fastGrowers-S_Vs_Alpha-contours}
  \caption{FGBTA}
  \label{subfig:FGBTA-alphagrad-contours}
 \end{subfigure}
\caption{Overall sizes of bacterial populations after 1000 time units have passed, for a variety of antibiotic gradients and initial nutrient concentrations.  
From this it can be seen that the SGBTA are less effective at inhibiting overall bacterial growth, however the impact of a uniform antibiotic concentration has opposing effects 
for SGBTA and FGBTA.  The presence of a gradient causes the SGBTA to be a more effective inhibitor, while the reverse is true for the FGBTA.}
\label{fig:alpha-popsize-contours}
\end{figure}


Some example results from these simulations are shown in Figures \ref{fig:const_C-popsize-contours}, \ref{fig:alpha-popsize-contours} and \ref{fig:alpha-spatdistbs}.  The first 
computational experiment performed entailed exposing the bacteria to a variety of nutrient and uniform antibiotic concentrations, as shown in Figure \ref{fig:const_C-popsize-contours}.  
This experiment, along with all the others, was initialised by setting $N_0 = 100$ and then allowing the system to develop for 1000 time units (increased to 2000 for the spatial 
distributions, to ensure the populations had reached their point of impedance).  As can clearly be seen by comparing Figures \ref{subfig:SGBTA-const_C-contours} and 
\ref{subfig:FGBTA-const_C-contours}, the SGBTA are much less effective at inhibiting bacterial growth than the FGBTA.

Following on from this, these simulations were then repeated but with the antibiotic applied in the form of an exponential gradient, with its steepness being varied.  Once 
again by comparing Figure \ref{subfig:SGBTA-alphagrad-contours} to \ref{subfig:FGBTA-alphagrad-contours}, it's observed that the SGBTA are still less effective at inhibiting 
bacterial growth when compared to the FGBTA.  However, the presence of a gradient compared has differing effects on the potency of the two antibiotic types.  By comparing Figures 
\ref{subfig:SGBTA-const_C-contours} and \ref{subfig:SGBTA-alphagrad-contours}, it can be seen that the presence of an antibiotic gradient causes the SGBTA to be relatively more 
effective at inhibiting the growth of the bacteria, while the reverse appears to be true for the FGBTA, as the presence of a gradient allows for more proliferation of bacteria.



\begin{figure}[H]
 \centering
 \begin{subfigure}[h]{0.3\textwidth}
 \includegraphics[width=\textwidth]{simple-slowGrowers-alpha=0_004884694070738408-spatialDistb}
  \caption{SGBTA}
  \label{subfig:SGBTA-spatdistb-spef_alpha}
  \end{subfigure}
  \begin{subfigure}[h]{0.3\textwidth}
   \includegraphics[width=\textwidth]{simple-flatGrowers-alpha=0_004884694070738408-spatialDistb}
   \caption{GIBTA}
   \label{subfig:GIBTA-spatdistb-spef_alpha}
  \end{subfigure}
  \begin{subfigure}[h]{0.3\textwidth}
  \includegraphics[width=\textwidth]{simple-fastGrowers-alpha=0_004884694070738408-spatialDistb}
  \caption{FGBTA}
  \label{subfig:FGBTA-spatdistb-spef_alpha}
 \end{subfigure}
\caption{Spatial distributions of bacterial populations exposed to SGBTA, GIBTA and FGBTA respectively.  The steepness of the antibiotic gradient has been chosen here such that 
the antibiotic concentration has a maximum of just over 10 in the final microhabitat.  This is done so that the largest possible value of the MIC is not reached until the far 
edge of the microhabitats, allowing for clear comparisons of the dependency on growth rate.  Once again, the SGBTA is inferior at inhibiting bacterial growth.}
\label{fig:alpha-spatdistbs}
\end{figure}


The objective of the next set of simulations performed was to determine of heterogeneous concentrations of these differing antibiotic types affected the spatial distribution 
of the bacteria, an example of which is shown in Figure \ref{fig:alpha-spatdistbs}.  Here we can once again see that the SGBTA allow for the bacteria to spread much further than the FGBTA 
in the same timeframe, with the GIBTA's efficacy being somewhere inbetween the two.  This shows that antibiotics which target the fast-growing tip of an advancing colony are more effective 
at impeding their progress.  

These simulations were then performed using more ``realistic'' expressions for the MICs and replication rates of relevant bacteria, from the findings in Greulich et al., 2015, which 
continue to corroborate these findings, i.e. that antibiotics which target fast-growing bacteria are more efficient at curtailing the spread of bacterial populations.  This component 
of the project has now deemed to have been completed and is currently in the process of being written up into a paper intended for publication.


\subsection{Multispecies models}

While many experimental investigations in laboratory conditions utilise biofilms containing only one species of microbe, in nature the vast majority of biofilms are actually composed 
of a wide variety of species, which have differing physical properties and typically are in competition with one another \cite{bioref:Elias-biofilm-multispecies-2012}.  Recently, I have 
moved onto a study that is motivated by our discussions with AkzoNobel, where they highlighted the importance of understanding how these multispecies biofilms behave in order to better 
design the combinations of antimicrobials they include in their anti-fouling paints.

To this end, work has now begun on a simple model to investigate the effect of these inter-species dynamics.  This design is 
currently in its infancy, and is once again based on the format of the simulations mentioned in the previous sections.  What sets these simulations apart from their predecessors is that 
the system is initialised with each microhabitat containing a number of randomly distributed microbial species.  Each of these species have differing levels of resistance to the applied 
antimicrobial, which is once again applied in the form of a gradient.  

Currently these resistance levels are simple numeric approximations which increase monotonically between each 
species, but there are plans for AkzoNobel to supply values for growth curves and other features of observed real-world organisms which have been determined via metagenomic analysis 
and other methods in their laboratories.  Additionally, discussions have been had about how best to incorporate another external source of bacteria to the system, 
representing the influx of microorganisms which one would experience in the wild.

\begin{figure}[H]
 \centering
 \includegraphics[height=5.8cm]{multispecies-snapshot}
 \caption{Snapshot of the GUI for the current multispecies model.  The most resistant species are shown at the bottom of the picture, with the resistances decreasing towards the top.  
 The system is exposed to an antibiotic gradient which is at its lowest on the left of the system, and increases exponentially towards the right.}
 \label{fig:multispecies-snapshot}
\end{figure}



\section{Proposal}

Over the next year, the first step will be to complete writing the paper on the growth-rate dependent antibiotic model.  All of
the results have now been collated, so a write-up and commentary, along with a discussion section is all that remains.  Along with some mainly stylistic editing.

Following on from this, work will continue on the multispecies model.  The current version is a simplistic toy model, similar to the one in the 
Greulich paper \cite{bioref:Greulich-growthDependentAntibiotics}.  Our industrial partners AkzoNobel have agreed to provide us with some more realistic parameters for factors such as 
growth and death rates for a variety of biofouling microorganisms, and their susceptibility to various biocides used in marine hull coatings.  When incorporated into the current model, 
this should provide us with some more accurate, if still somewhat simplified, results for how a system with multiple species compete with each other over time.  AkzoNobel themselves 
have also monitored species composition over time on a variety of antimicrobial surfaces, so this should prove useful for comparison.  I estimate 
that at least, but no more than a few months will be spent working on this model.

While these additions should certainly yield some interesting results, there is only so far that these simplistic models can take us in providing a clear picture of the mechanics 
involved in how biofilms respond to the application of antibiotics.  Therefore the plan at present is to eventually move on from this 1D ``lattice'' model and begin work on a more 
intricate continuum model.  This model would allow us to incorporate key features of biofilms and the environment which surrounds them, such as surface roughness and flow around 
the biofilm.  These models should allow us to perform research on both the medical and industrial scenarios where biofilms form, by varying the side of the biofilm that the 
drug gradient arises from, and discerning whether these two situations create differing outcomes.  I predict that completing the existing multispecies model and setting up the more 
advanced simulations will consume the majority of the second year of my PhD.



\begin{figure}[H]
 \centering
 \includegraphics[width=\textwidth]{Matsumoto-biofilm-simulation}
 \caption{Screenshot taken from the paper by Matsumoto at al., 2007 \cite{bioref:Matsumoto-sim-snapshot-2007} which contained several differing species of bacteria whose growth rates 
 varied depending on the atmospheric composition of the biofilm.  This is included for illustrative purposes as an idea of what the simulations constructed in the later 
 stages of this PhD project might resemble.}
 \label{fig:Matsumoto-sim-snapshot}
\end{figure}

In conjunction with AkzoNobel, some wet work will also be undertaken by me at their laboratory in Newcastle.  This would entail cultivating biofilms on various surfaces akin to industrial 
ship hulls and exposing them to a variety of biocidal paints and then examining the composition of the established biofilm to compare the model with reality.  There might also 
be opportunity to undertake part of the metagenomic analysis of these cultivated biofilms to determine species composition.  I think this will be helpful to the project as it will 
complement the simulations, but is not intended as an integral component of my PhD.  It should only require a month or so of involvement, and will likely be performed alongside my 
other commitments. 

% In this section, detail, as far as you are currently able, 
% your research plan for the second and third years of your PhD, 
% drawing from the key references \cite{jr:block} that you have highlighted in your review section. 
% Here, try and illustrate
% your proposal, as in Figure~\ref{fig:prism} which is taken from the
% same paper as the illustration references.
% 
% 
% At this stage it is {\em not} expected that this will be a fully-developed
% research proposal, but is your chance to show what you have extracted from the
% literature and how you see your own work will fit in. This section is
% not expected to exceed 2 pages.
 
\section{Summary}

Although substantial research has been undertaken in the field of antibiotic and antimicrobial resistance, many of these studies were performed under idealised conditions, and neglect 
important environmental factors, such as the presence of a heterogeneous antimicrobial concentration gradient, in lieu of a uniform concentration which is typically used in laboratory 
environments.  To this end, simple models investigating the effects of antibiotic gradients have been constructed which demonstrate the relation between gradient steepness and time taken 
for resistance to emerge, the differing inhibition properties of growth-rate dependent antibiotics and how colonies involving multiple species with differing levels of resistance 
develop over time.  An article is currently being written on the findings of the growth-rate dependent simulations.

The aim for the next year or so is to finish writing the article regarding the work on the growth-dependent antibiotics, then to continue with the multispecies
model and hopefully incorporate some realistic parameters contributed by our industrial partners AkzoNobel.  Following on from this, work will begin on the proposed
continuum model which could incorporate other factors such as surface texture and flow into the system, which would allow us to perform more realistic biofilm simulations as a 
long-term goal.  

This will be accompanied by some wet work at the AkzoNobel laboratory
at their Newcastle compound.  There has also been some discussion of
my involvement in AkzoNobel's metagenomic analysis of organisms gathered from their laboratory to extract information about the growth rates and biocidal susceptibility from 
microorganisms taken from biofilms attached to industrial shipping vessels.


\pagebreak
\newpage
%            Now build the reference list
\bibliographystyle{ieeetr}                      % The reference style
%                This is plain and unsorted, so in the order
%                they appear in the document.


\bibliography{BiofilmReferences}       % Multiple bib files.

\end{document}

